tests/src/Kernel/SmsFrameworkProviderTest.php

```diff
@@ -383,6 +388,7 @@ public function testEventsQueueIncoming() {
     $this->container->get('cron')->run();
 
     $expected[] = SmsEvents::MESSAGE_INCOMING_PRE_PROCESS;
+    $expected[] = 'incomingEvent'; // Plugin incoming event.
```

Comment:
> I think we should use a constant like the others for consistency sake. Even though it's not a subscribed Symfony event like the others. The purpose of a constant is to store a value so you remain DRY in your programming.

Except this value 'incomingEvent' is ONLY used for testing, and only in the sms_test_gateway submodule. The events are used in normal operation of the module.

The 'incomingEvent' is almost constant-like in its meaning, because it is derived from the method name. It isnt set from a plain string. See `\Drupal\sms_test_gateway\Plugin\SmsGateway\Memory::incomingEvent`